linode_api4/linode_client.py

```diff
@@ -402,6 +402,127 @@ def __call__(self):
         p = Profile(self.client, result["username"], result)
         return p
 
+    def trusted_devices(self):
+        """
+        Returns the Trusted Devices on your profile.
+
+        API Documentation: https://www.linode.com/docs/api/profile/#trusted-devices-list
+
+        :returns: A list of Trusted Devices for this profile.
+        :rtype: PaginatedList of TrustedDevice
+        """
+        return self.client._get_and_filter(TrustedDevice)
+
+    def user_preferences(self):
+        """
+        View a list of user preferences tied to the OAuth client that generated the token making the request.
+        """
+
+        result = self.client.get(
+            "{}/preferences".format(Profile.api_endpoint), model=self
+        )
+
+        return MappedObject(**result)
+
+    def security_questions(self):
+        """
+        Returns a collection of security questions and their responses, if any, for your User Profile.
+        """
+
+        result = self.client.get(
+            "{}/security-questions".format(Profile.api_endpoint), model=self
+        )
+
+        return MappedObject(**result)
```

Comment:
Could we also cast all security question objects in the response to mapped objects?